### ISSUE TITLE
some questions regarding the accuracies 

### ISSUE THREAD

**Issue by imaxpayne** [floodsung/LearningToCompare_FSL#3]
Hi, I read both old and new version of the paper and found that the "shallow" RN in new version of the paper has lower accuracy than the one in older version of the paper, despite the number of layers and number of filters in each layer are exactly the same, what makes the accuracy become lower?

Also, In the older version of the paper, it report the "deeper" RN with better performance, and stated that the model can benefit from a deeper structure, but when I added only 2 additional conv layer on embedding part of the model and trained it  on mini-imagenet, it was overfitted after about 80000 episode and get poor test accuracy which is below 0.60, Am I missing any trick to make it deeper?
